manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -4,7 +4,11 @@ metadata:
   name: ztunnel
   namespace: {{ .Release.Namespace }}
   labels:
-    {{- .Values.labels | toYaml | nindent 4}}
+    istio.io/component: ztunnel
+    operator.istio.io/component: Ztunnel
+    k8s-app: ztunnel
+    app: ztunnel
+{{ with .Values.labels -}}{{ toYaml . | nindent 4}}{{ end }}
```

Comment:
This is fine for now, but there is zero reason we can't do this as a template for all charts in exactly the same way, such that we never have to worry about this problem again, e.g.

https://github.com/istio/istio/issues/52034